lib/middlewares/racker.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
+
+require 'erb'
+require 'codebreaker/game'
+
+module Middlewares
+  # Main controller class
+  class Racker
+    def self.call(env)
+      new(env).response.finish
+    end
+
+    def initialize(env)
+      @request = Rack::Request.new(env)
+      @game = Codebreaker::Game.new
```

Comment:
Looks like u use it only for new games. Move it to `new_game`

Reply:
No, @game used also to view statistics which can be called without starting new game